wlog/context_default_test.go

```diff
@@ -172,10 +172,11 @@ func testFromContextFromEmptyContextForSingleLogger(t *testing.T, tmpDir string,
 			verify: func(loggerTestCaseInfo loggerTestCase, bytes []byte) {
 				logOutput := string(bytes)
 
-				firstPortionRegexp := regexp.MustCompile(
-					regexp.QuoteMeta(`[WARNING] github.com/palantir/witchcraft-go-logging/wlog_test.TestOutputFromContextEmptyContext`) + ".+" + regexp.QuoteMeta(`/github.com/palantir/witchcraft-go-logging/wlog/context_default_test.go:`) + "[0-9]+" + regexp.QuoteMeta(`]: usage of `+loggerTestCaseInfo.loggerPkg+`.Logger from FromContext that did not have that logger set: `))
+				firstPortionRegexp, err := regexp.Compile(
+					regexp.QuoteMeta(`[WARNING]`) + ".*" + regexp.QuoteMeta(`github.com/`) + ".+" + regexp.QuoteMeta(`/witchcraft-go-logging/wlog_test.TestOutputFromContextEmptyContext`) + ".+" + regexp.QuoteMeta(`/github.com/`) + ".+" + regexp.QuoteMeta(`/witchcraft-go-logging/wlog/context_default_test.go:`) + "[0-9]+" + regexp.QuoteMeta(`]: usage of `+loggerTestCaseInfo.loggerPkg+`.Logger from FromContext that did not have that logger set: `))
+				require.NoError(t, err, "Unexpected error compiling regex")
```

Comment:
This was modified so that people testing on a fork do not look for the specific path that depends on not being a fork. i.e. `github.com/[^/]+/witchcraft-go-logging` is valid.